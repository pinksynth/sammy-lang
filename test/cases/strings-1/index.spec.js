const {
  expectCompiledEval,
  performSnapshotAssertions,
} = require("../../helpers")
performSnapshotAssertions("Strings", __dirname)

test("Strings implementation", () => {
  expectCompiledEval(`"Foo"`).toBe("Foo")
})
