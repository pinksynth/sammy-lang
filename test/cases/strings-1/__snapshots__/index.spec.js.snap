// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ“¸ Strings 1`] = `
[
  {
    "columnNumberEnd": 12,
    "columnNumberStart": 1,
    "lineNumberEnd": 1,
    "lineNumberStart": 1,
    "tokenType": "STRING_PURE",
    "value": ""Fooberton"",
  },
]
`;

exports[`ðŸ“¸ Strings 2`] = `
{
  "children": [
    {
      "columnNumberStart": 1,
      "interpolations": [],
      "lineNumberStart": 1,
      "subStrings": [
        "Fooberton",
      ],
      "type": "LITERAL_STRING",
    },
  ],
  "type": "ROOT",
}
`;

exports[`ðŸ“¸ Strings 3`] = `"\`Fooberton\`"`;
